content/docs/manuals/extending-oakestra/install_addon.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Install an Addon"
+summary: ""
+draft: false
+weight: 2
+toc: true
+seo:
+  title: "" # custom title (optional)
+  description: "" # custom description (recommended)
+  canonical: "" # custom canonical URL (optional)
+  noindex: false # false (default) or true
+---
+
+## Introduction
+
+Addons in Oakestra allow developers to extend the platform by adding new features or enhancing existing ones. This guide focuses on the installation process for addons, enabling you to integrate published addons into your Oakestra environment seamlessly.
+
+
+## Install an Addon
+
+{{< callout context="note" title="Note" icon="outline/info-circle" >}}
+Before installing an Addon make sure that the Addons System is running and that the Addon to be installed is published in the Marketplace.
+
+- Verify that the Addon is published in the Marketplace. This can be done by sending a `GET` request to `/api/v1/marketplace/addons/{addon_marketplace_id}`.
+- Also verify that the Addons Engine component is running inside your orchestrator.
```

Comment:
How to verify this? Give the command or point to specific pointer in setup that someone should lookout for